Gemfile

```diff
@@ -37,3 +37,5 @@ group :test do
   gem 'rubocop-rails', require: false
   gem 'rubocop-rspec', require: false
 end
+
+gem 'iex-ruby-client'
```

Comment:
can you put it on the upper part of the Gemfile?

Reply:
resposition gem 'iex-ruby-client'